tests/cloudpickle_test.py

```diff
@@ -78,6 +78,7 @@ def setUp(self):
 class CloudPickleTest(unittest.TestCase):
 
     protocol = cloudpickle.DEFAULT_PROTOCOL
+    override_existing_globals = False
```

Comment:
Why do you use a class variable here?

Reply:
To parametrize the tests. `Protocol2CloudPickleTest` uses `override_existing_globals = True`.